cli/batch/forms.py

```diff
@@ -0,0 +1,18 @@
+"""
+Provide forms for command line interface's batch commands.
+"""
+from marshmallow import Schema
+
+from cli.generic.forms.fields import (
+    BatchIdentifierField,
+    NodeUrlField,
+)
+
+
+class GetBatchForm(Schema):
+    """
+    Get a batch by its identifier form.
+    """
+
+    id = BatchIdentifierField(allow_none=True, required=True)
```

Comment:
Why you allow none here if the identifier is required for using the command?

Reply:
Done.